utopia-remix/package.json

```diff
@@ -93,6 +94,7 @@
     "eslint-plugin-prettier": "3.1.2",
     "eslint-plugin-react": "7.33.2",
     "eslint-plugin-react-hooks": "4.6.0",
+    "eslint-plugin-react-refresh": "0.4.12",
```

Comment:
Interesting. I'm guessing this wasn't intentionally committed since it hasn't been added to our eslint config? It looks like it would be useful to us, so worth bringing up on Discord (unless you already did and I missed it!)

Reply:
so this is the plugin that we discussed about a month ago that prevents accidentally breaking the HMR. Balazs was adding it to the eslint config and to the editor, but we needed to install it in the `utopia-remix` as well, otherwise the eslint didn't work at all.